### PR TITLE
feat: `cacheKeys` option

### PR DESCRIPTION
**Pull request by evilebottnawi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,7 @@ module.exports = {
 |**`include`**|`{RegExp\|Array<RegExp>}`|`undefined`|Files to `include`|
 |**`exclude`**|`{RegExp\|Array<RegExp>}`|`undefined`|Files to `exclude`|
 |**`cache`**|`{Boolean\|String}`|`false`|Enable file caching|
+|**`cacheKeys`**|`{Function(defaultCacheKeys, file) -> {Object}}`|`defaultCacheKeys => defaultCacheKeys`|Allows you to override default cache keys|
```
Comment (ooflorent) [inline README.md]: What about the 3rd param?

Comment (shellscape) [reply README.md]: seconded

Comment (evilebottnawi) [reply README.md]: Don't understand :smile:

Comment (shellscape) [reply README.md]: @evilebottnawi sorry. that means "I agree, and have the same comment"

Comment (evilebottnawi) [reply README.md]: @shellscape i don't understand what i do :smile: 3rd param? What is mean?

Comment (evilebottnawi) [reply README.md]: /cc @ooflorent

Comment (ooflorent) [reply README.md]: Example below:

```js
[
  new UglifyJsPlugin({
    cache: true,
    cacheKeys: (defaultCacheKeys, file, options) => {
      //                                ^^^^^^^
      defaultCacheKeys.myCacheKey = 'myCacheKeyValue';
      
      return defaultCacheKeys;
    },
  })
]
```

I now understand that my comment was wrong. The signature is correct, the example is not.

File: README.md [hunk 2]
```diff
@@ -112,6 +113,35 @@ Default path to cache directory: `node_modules/.cache/uglifyjs-webpack-plugin`.
...
 Path to cache directory.
 
+### `cacheKeys`
+
+**webpack.config.js**
+```js
+[
+  new UglifyJsPlugin({
+    cache: true,
+    cacheKeys: (defaultCacheKeys, file, options) => {
```
Comment (ooflorent) [inline README.md]: Remove the 3rd argument please

Comment (evilebottnawi) [reply README.md]: /cc @ooflorent done, need approve :smile:

**Pull request merged**